### DISCUSSION TITLE
QR Questions

### DISCUSSION THREAD

**Discussion by samcraignyc** [gopro/labs#235] [Q&A]
Hi all. I searched but didn't see answers to these questions. Hoping someone can help..

1. It looks like currently, there is only a way to "overwrite"/create a QR code for default/existing video presets (standard, activity, cinematic, etc).  Is this correct? IE - you can't create one for say, "Water."
2. I had my preset of water (video) as active, then scanned a QR code with different settings, and pretty sure it wrote over my water settings, not standard.  Was that a glitch? It appears that whatever preset I have selected, that's what the QR effects?
3. If this is the case, I suppose I would need to create a qr code with my current "water" settings, to always bring that back to my personal default?
4. Are there plans to add QR codes that can be assigned a name on the menu?

Thanks!

**Comment by dnewman-gpsw** [comment]
1. Yes
2. If you don't specific which preset to modify it changes the current one, which the default and correct operation.
3. Yes
4. Not until the main firmware supports it.  Labs doesn't alter the UI, and this would be a UI change.
